release/models/interfaces/openconfig-interfaces.yang

```diff
@@ -197,6 +204,16 @@ module openconfig-interfaces {
       name and subinterface index.";
   }
 
+  typedef interface-full-id {
+    type string {
+      pattern '(.*)\.([0-9]*)';
+      oc-ext:posix-pattern '^(.*)\.([0-9]*)$';
```

Comment:
Should this be `(.+)\.([0-9]+)` to avoid `.` as a valid interface id?

Reply:
Yes! Done.